apps/services/auth/public-api/src/app/modules/delegations/actorDelegations.controller.ts

```diff
@@ -88,6 +89,10 @@ export class ActorDelegationsController {
       )
     }
 
-    return this.delegationsService.findAllIncoming(actor, delegationTypes)
+    return (
+      await this.delegationsService.findAllIncoming(actor, delegationTypes)
+    ).filter(
+      (d) => d.type != DelegationType.Custom || d.domainName == DEFAULT_DOMAIN,
+    )
```

Comment:
Let's keep it like this for now, but this (public) API should support delegations for other domains as well (through the domain of the authenticated client, or fancy delegation merging).

We do plan to eventually move the meDelegations controller to an internal API but keep this one public for other orgs, so hopefully we can fix this soon.